### ISSUE TITLE
Aria-Labelledby on heading ignores name from image with aria-hidden true

### ISSUE THREAD

**Issue by LaurenceRLewis** [FreedomScientific/standards-support#777]
### Summary
> HTML heading has aria-labelledby and id.
> Image has aria-hidden set to true and ID.
> Aria-labelledby on the heading is pointing to the ID of the image and the ID of the heading. 

### Expected result

> JAWS announces: [Heading Level] (Image name) (Heading name).

### Actual result

> JAWS announces: [Heading Level] (Heading name)

### Example

> A test case that demonstrates the issue
- [Testing aria-labelledby on heading | JAWS 2023.2307.37 bug](https://cdpn.io/pen/debug/PoVNXxY)
#### How to test

1.  Run JAWS
2. Navigate to the heading

### Additional Information
Works as expected running NVDA, VoiceOver and TalkBack.
Full name is exposed in the Accessibility Tree

#### JAWS version and build number
- JAWS 2023.2307.37

 
#### Operating System and version
- WIndows 10 Pro

#### Browser and version:
- Chrome 119
- Edge 118
- Firefox 119

**Comment by JAWS-test** [comment]
The problem has nothing to do with SVG and aria-hidden and not even with aria-labelledby. It is just a general problem: JAWS always ignores the Accessible Name of a heading and uses only the text content. This can be easily checked by using an aria-label on a heading. I suggest that you (@LaurenceRLewis) adjust the title of this issue to make it clear that this is a general problem and not a specific one

**Comment by LaurenceRLewis** [comment]
@JAWS-test If this is a JAWS specific behaviour, that is different to every other screen reader tested, and the name from the SVG and the text node of the heading via aria-labelledby is available in the accessibility tree in the browsers. How is this a general issue?

**Comment by LaurenceRLewis** [comment]
@JAWS-test Title updated.

**Comment by JAWS-test** [comment]
> JAWS is still not speaking the information available in the accessibility tree of the browsers

Yes, that is exactly the problem. But your title suggests that the problem only occurs with SVG with aria-hidden. This makes the problem smaller and more specific than it is. The problem always occurs, i.e. JAWS always ignores the Accessible Name of a heading. This is a very serious and general problem of JAWS. The title should be "JAWS ignores the Accessible Name of headings".

**Comment by stevefaulkner** [comment]
@BrettLewisVispero I can reproduce this in JAWS 2024. test case: https://cdpn.io/pen/debug/PoVbXLd

1. heading with aria-labelledby
2. heading with aria-label

In both cases JAWS ignores the accessible name provided via ARIA and reads only the on screen text.

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 49061

**Comment by LaurenceRLewis** [comment]
@stevefaulkner I discovered the bug when testing a pattern in our design system. I have not looked for, or seen it in the wild.

**Comment by JAWS-test** [comment]
I have come to believe that JAWS is doing everything right here and that the ticket should be closed. JAWS is deliberately deviating from the ARIA specification in this case, but this makes sense because the ARIA specification is [flawed](https://github.com/w3c/aria/issues/2797). If JAWS were to adhere to the ARIA specification, we would face serious problems similar to those with NVDA (https://github.com/nvaccess/nvda/issues/13296).
